cmd/internal/find/types/options.go

```diff
@@ -49,16 +52,58 @@ func (opts *Options) MarkAsSet(flag string) {
 	opts.setFlags[flag] = struct{}{}
 }
 
+// HelpRequested returns true if the help option was set,
+// false otherwise.
+func (opts *Options) HelpRequested() bool {
+	return opts.IsSet("help") || opts.IsSet("h")
+}
+
 // FlagSet returns a flagset representing
 // opts.
 func (opts *Options) FlagSet() *flag.FlagSet {
 	// Create the flag-set that's tied to our options.
 	fs := flag.NewFlagSet("options", flag.ContinueOnError)
-	// TODO: Handle usage later
-	fs.Usage = func() {}
 	fs.SetOutput(ioutil.Discard)
 	fs.BoolVar(&opts.Depth, DepthFlag, opts.Depth, "")
 	fs.UintVar(&opts.Mindepth, MindepthFlag, opts.Mindepth, "")
 	fs.IntVar(&opts.Maxdepth, MaxdepthFlag, opts.Maxdepth, "")
+	fs.Var(&opts.Help, "help", "")
+	fs.Var(&opts.Help, "h", "")
 	return fs
 }
+
+// OptionsTable returns a table containing all of `wash find`'s available
+// options
+func OptionsTable() *cmdutil.Table {
+	return cmdutil.NewTable(
+		[]string{"Flags:",                 ""},
+		[]string{"      -depth",           "Visit the children first before the parent (default false)"},
+		[]string{"      -mindepth levels", "Do not print entries at levels less than levels (default 0)"},
+		[]string{"      -maxdepth levels", "Do not print entries at levels greater than levels (default infinity)"},
+		[]string{"  -h, -help",            "Print the usage"},
+		[]string{"  -h, -help primary",    "Print a detailed overview of the specified primary"},
+		// TODO: Is this the right way to distinguish string literals?
+		[]string{"  -h, -help 'syntax'",   "Print a detailed overview of find's expression syntax"},
```

Comment:
`syntax` is meant to be literal, i.e. the user's supposed to pass in `-h syntax`. Is quoting it the correct way to indicate that in the usage?

Reply:
I think the common way is to use `wash help syntax` and `wash help <primary>`. Also, `-help` with only a single dash seems odd.

Reply:
Single dash is to make the syntax consistent with the other options + the expression operators/primaries.